docs/resources/server.md

```diff
@@ -22,6 +22,7 @@ data "ionoscloud_image" "example" {
     type                  = "HDD"
     cloud_init            = "V1"
     image_alias           = "ubuntu:latest"
+    location              = "us/ewr"
```

Comment:
shouldn't this match the datacenter location? us/las?

Reply:
@cristiGuranIonos no, it doesn't matter where the image is coming from.

Reply:
what if it's not available in the location?

Reply:
I mean, if it's an example we should keep the location the same in the entire example, even if ubuntu:latest is available in all locations.

Reply:
@cristiGuranIonos `what if it's not available in the location?` then the image will not be fetched, I don't think that we can reach a configuration to ensure that a image will always be available, whether we talk about `location`, `image_alias` or `type`. 

`if it's an example we should keep the location the same in the entire example, even if ubuntu:latest is available in all locations.` ok, I will change the location

Reply:
@cristiGuranIonos done